nautobot/extras/models/datasources.py

```diff
@@ -120,6 +120,15 @@ def token_rendered(self):
     def filesystem_path(self):
         return os.path.join(settings.GIT_ROOT, self.slug)
 
+    def set_dryrun(self, init_repo=False):
+        """
+        Add _init_repo to self
+
+        Args:
+            init_repo (str): Mode of dry run either create/update
+        """
```

Comment:
Would this be correct?

```suggestion
        """
        Add _init_repo flag.
        
        The existence of this flag indicates that the next sync of this repo (on save()) should be a dry-run only;
        the value of this flag indicates whether the repo will need to be initialized as empty before the dry-run

        Args:
            init_repo (bool): True if the repo needs to be initialized as empty
        """
```

Reply:
Perfect, Thanks